src/main/java/com/auth0/client/mgmt/ManagementAPI.java

```diff
@@ -15,16 +15,17 @@
 public class ManagementAPI {
 
     private final HttpUrl baseUrl;
-    private final String apiToken;
+    private String apiToken;
```

Comment:
I don't know if `ManagementAPI` is supposed to be thread safe, if yes then `apiToken` should have `volatile` modifier.